include/z64dma.h

```diff
@@ -16,24 +28,11 @@ typedef struct {
 } DmaRequest; // size = 0x20
 
 typedef struct {
-    /* 0x00 */ uintptr_t vromStart;
-    /* 0x04 */ uintptr_t vromEnd;
+    /* 0x00 */ RomFile file;
     /* 0x08 */ uintptr_t romStart;
     /* 0x0C */ uintptr_t romEnd;
 } DmaEntry;
```

Comment:
interesting, the conservative in me doesn't like touching such a fundamental struct but I appreciate that it links together the RomFile (vrom) references and the physical rom

Reply:
I think what I don't like about it is that when it's used here, it's obvious that the RomFile use case is too narrow. `vromStart`/`vromEnd` should be the same type of thing as `romStart`/`romEnd`